boards/arm/nrf9131ek_nrf9131/Kconfig.board

```diff
@@ -0,0 +1,14 @@
+# nRF9161 DK NRF9161 board configuration
+
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF9161_SICA
```

Comment:
should there be a new SOC also?

Reply:
good question! technically, it shouldn't be necessary, but it could be for documentation reasons. we could also nename nRF9161 to nRF91X1. Who would decide this?